thetis/options.py

```diff
@@ -459,6 +459,11 @@ class CommonModelOptions(FrozenConfigurable):
 
         Bottom stress is :math:`\tau_b/\rho_0 = -g \mu^2 |\mathbf{u}|\mathbf{u}/H^{1/3}`
         """).tag(config=True)
+    norm_smoother = FiredrakeConstantTraitlet(
+        Constant(0.0), help=r"""
+        Coefficient used to avoid non-differentiable functions in the continuous formulation of the velocity norm in
```

Comment:
It would be helpful to add the formula from the paper, something like:
```
This replaces the velocity norm in the quadratic bottom drag term with :math:`\|u\| \approx \sqrt{\|u\|^2 + \alpha^2}`
```
so people have an idea what sort of value to put, i.e. a *small* velocity with units of m/s.